gormsupport/cleaner/db_clean.go

```diff
@@ -46,9 +50,14 @@ func DeleteCreatedEntities(db *gorm.DB) func() {
 		tx := db.Begin()
 		for i := len(entires) - 1; i >= 0; i-- {
 			entry := entires[i]
-			fmt.Println("Deleting from", entry.table, entry.key)
+			log.Println("Deleting from", entry.table, entry.key)
 			tx.Table(entry.table).Where(entry.keyname+" = ?", entry.key).Delete("")
 		}
+
+		// Delete the work item cache as well
+		// NOTE: Feel free to add more cache freeing calls here as needed.
+		workitem.NewWorkItemTypeRepository(tx).ClearCache()
```

Comment:
Because ClearCache() is a method not a function. You need a WIT repo instance to call it. But it clears the global cache (map) so it will work.